### PR TITLE
Update links to reflect new JuliaGeometry ownership

### PR DESCRIPTION
**Pull request by JoshuaLampert**
After moving the repo to the JuliaGeometry org, the link for the documentation should be adjusted accordingly.
This is also a test PR to see whether CI, codecov and documentation deployment works.

### CODE REVIEW

**Comment by JoshuaLampert** [comment]
Deploying the preview of the documentation works and is also hosted at the correct URL, see https://juliageometry.github.io/MeshIntegrals.jl/previews/PR117/?pr=117.

File: src/MeshIntegrals.jl [hunk 1]
```diff
@@ -24,6 +24,7 @@ include("integral_aliases.jl")
 export lineintegral, surfaceintegral, volumeintegral
 
 # Integration methods specialized for particular geometries
+
```
Comment (JoshuaLampert) [inline src/MeshIntegrals.jl:27]: This is just a test to trigger CI and see if everything looks fine with CI and code coverage after the move.
```suggestion
```

**Comment by JoshuaLampert** [comment]
Looking at https://coveralls.io/github/JuliaGeometry/MeshIntegrals.jl seems like coveralls successfully recognizes the move, but https://app.codecov.io/gh/JuliaGeometry/MeshIntegrals.jl seems like codecov didn't. That something with codecov is still not working is also reflecting by the missing comment from the codecov bot in the PR. But to investigate further we would need to look into the settings of the repo and check the CODECOV_TOKEN (maybe it changed after the move and needs to be updated with the new one?), however this is currently pending https://github.com/JuliaGeometry/MeshIntegrals.jl/issues/115#issuecomment-2432810216

**Comment by JoshuaLampert** [comment]
We also need to remember to update the link to the docs in the about section of the repo, but I guess you can also not edit this one @mikeingold, right?

**Comment by mikeingold** [comment]
> We also need to remember to update the link to the docs in the about section of the repo, but I guess you can also not edit this one @mikeingold, right?

Correct. That’s actually what made me realize I had lost owner rights, since I now don’t have access to the settings icon that I would’ve used to modify that link.

**Comment by JoshuaLampert** [comment]
> > We also need to remember to update the link to the docs in the about section of the repo, but I guess you can also not edit this one @mikeingold, right?

> Correct. That’s actually what made me realize I had lost owner rights, since I now don’t have access to the settings icon that I would’ve used to modify that link.

Ok, let's wait for Julios response.

**Comment by juliohm** [comment]
@JoshuaLampert @mikeingold you should both have admin rights in MeshIntegrals.jl now. Please let me know if it is not the case.

**Comment by mikeingold** [comment]
When I pull up my CodeCov profile I can still see the repo as if it's `mikeingold/MeshIntegrals.jl`. I don't see any options to update the source/url of the repo and the last commit it recognizes was 2 days ago, so it seems to have gotten disconnected.

Once we get owner rights set up, it looks like we can just use [this link](https://app.codecov.io/gh/JuliaGeometry/MeshIntegrals.jl/new) to set it up again from scratch with a new token.

**Comment by mikeingold** [comment]
> We also need to remember to update the link to the docs in the about section of the repo

Done.

**Comment by mikeingold** [comment]
CodeCov token updated.

**Comment by JoshuaLampert** [comment]
I reran CI. Let's see what happens.

**Comment by JoshuaLampert** [comment]
Looks like everything we are all set👍

**Review by mikeingold — APPROVED** [review]

**Pull request merged**